pkg/api/rpms.go

```diff
@@ -105,6 +105,8 @@ type ModuleInfoResponse struct {
 	Version      string         `json:"version,omitempty"`     // Version of the module
 	Description  string         `json:"description,omitempty"` // Description of the module
 	PackageNames pq.StringArray `json:"-" gorm:"type:text"`    // Packages included in the module
+	StartDate    string         `json:"start_date"`
```

Comment:
omitting might be a good idea, also i can't remember from the diagram.. should we be returning the dates if it's not a module? i guess omitting if empty will handle this if we shouldn't be

```
{
        "package_name": "389-ds-base",
        "summary": "389 Directory Server (base)",
        "package_sources": [
            {
                "type": "package",
                "start_date": "",
                "end_date": ""
            }
        ]
    },
```

Reply:
I believe we would also return it for packages, although that is for the next task